src/components/task-plan/homework.cjsx

```diff
@@ -184,19 +184,36 @@ AddExercises = React.createClass
 
 
 ExerciseSummary = React.createClass
+  addTutorSelection: ->
+    TaskPlanActions.updateTutorSelection(@props.planId, 1)
+
+  removeTutorSelection: ->
+    TaskPlanActions.updateTutorSelection(@props.planId, -1)
+
   render: ->
     if not @props.shouldShow
       return <span></span>
 
     numSelected = TaskPlanStore.getExercises(@props.planId).length
-    total = numSelected + 3
+    numTutor = TaskPlanStore.getTutorSelections(@props.planId)
+    total = numSelected + numTutor
 
     if @props.canReview and numSelected
       button = <BS.Button bsStyle="primary" onClick={@props.reviewClicked}>Review</BS.Button>
     else if @props.canAdd
       button = <BS.Button bsStyle="primary" onClick={@props.addClicked}>Add</BS.Button>
 
+    if numTutor > TaskPlanStore.getTutorMinimum()
```

Comment:
Instead of `getTutorMinimum` you could have `canDecreaseTutorExercises` to not have to expose the min/max explicitly.